src/stan_math_backend/Expression_gen.ml

```diff
@@ -300,6 +308,10 @@ and gen_fun_app ppf fname es =
         , "integrate_ode_rk45"
         , f :: y0 :: t0 :: ts :: theta :: x :: x_int :: tl ) ->
           (fname, f :: y0 :: t0 :: ts :: theta :: x :: x_int :: msgs :: tl)
+      | ( true
+        , x
+        , {pattern= FunApp (_, f, _); _} :: grainsize :: container :: tl ) when List.mem ~equal:(String.equal) Stan_math_signatures.reduce_sum_functions x->
```

Comment:
This could use some line breaks. It's kind of hard to review code that doesn't fit on the screen. I'm not saying you should manually format code though. Personally, I  have set up an editor hotkey that runs `ocamlformat` and I always hit that instead of enter.

Reply:
I typically run `dune build @fmt` which is run with `make format`. I think I forgot to run it here.